ihmc-perception/src/main/java/us/ihmc/perception/detections/yolo/YOLOv8DetectionExecutor.java

```diff
@@ -63,14 +63,14 @@ public class YOLOv8DetectionExecutor
 
    // TODO: temp hack
    private int lastRunDetectorIndex = 0;
-   private final Map<Integer, YOLOv8DetectionResults> yoloDetectionResults = new HashMap<>();
+   private final Map<Integer, YOLOv8DetectionResults> yoloDetectionResults = new ConcurrentHashMap<>();
```

Comment:
Not sure if this is necessary but technically this could be accessed concurrently so I thought it'd be safer this way.